controller/src/test/java/io/pravega/controller/server/v1/ControllerServiceImplTest.java

```diff
@@ -309,11 +309,16 @@ public void testCreateStreamThrowsLockFailed() {
         // Check that concurrent calls to create a stream throw FailedLockingException
         final ScalingPolicy policy = ScalingPolicy.fixed(2);
         final StreamConfiguration configuration = StreamConfiguration.builder().scalingPolicy(policy).build();
+
+        ResultObserver<CreateScopeStatus> result = new ResultObserver<>();
+        this.controllerService.createScope(ScopeInfo.newBuilder().setScope("locking-scope").build(), result);
```

Comment:
Minor: please, make the scope name a variable to be used throughout the test case.